src/app/onboarding/view.nim

```diff
@@ -40,6 +40,11 @@ QtObject:
     self.beginInsertRows(newQModelIndex(), self.accounts.len, self.accounts.len)
     self.accounts.add(account)
     self.endInsertRows()
+  
+  proc removeAccounts*(self: OnboardingView) =
+    self.beginRemoveRows(newQModelIndex(), self.accounts.len, self.accounts.len)
```

Comment:
depending on the amount of data in self.accounts, it might be cheaper to do a `beginResetModel()` and `endResetModel()`